lib/helpers/search-results-to-template-data/title-page.js

```diff
@@ -14,9 +14,11 @@ var nameFromFilter = function (selectedFilters) {
   var museum = selectedFilters.museum && Object.keys(selectedFilters.museum)[0];
   var gallery = selectedFilters.gallery && Object.keys(selectedFilters.gallery)[0];
 
+  var exclude = ['categories', 'museum', 'gallery', 'type', 'page[size]', 'page[type]'];
+
   // check that only the filter categories, museum or gallery are selected
```

Comment:
No, not really. The functionality hasn't changed, it was just a bug that was picking up default seetings such as page type and size